### ISSUE TITLE
[ADE Intake] VA Health and Benefits App - Accessibility review of VetRide (web and native mobile)

### ISSUE THREAD

**Issue by artsymartha68** [department-of-veterans-affairs/va.gov-team#122513]
> [!NOTE]
> **QUICK SUMMARY:** _ADE accessibility review of VetRide web site and native mobile app_

## Description of the Work

Requesting standard ADE accessibility evaluation process for VetRide web site and native mobile app. VetRide is a VA product but this is similar to recent COTS product evaluations produced by ADE. 

### Background

VetRide is a service for Veterans to get transportation to and from their VA appointments. Blinded Veterans Association members have suggested the idea of integrating VetRide with VAHB. Transportation is a major issue for this population. 

VAHB is investigating the idea of launching VetRide from within the app. We all want to make sure that all the VetRide pieces are accessible to OCTO's "accessibility beyond compliance" standards. Per @Logan-Smith1 VetRide is already 508 compliant, which is great to hear!

ADE, please work with @Logan-Smith1 (he's in slack) to coordinate access, test accounts, etc. 

## About the Team 

- **Team name**: VAHB
- **Product name:** VetRide
- **VetRide product owners:**  @Logan-Smith1 and [Johnathon.CobleHarrison@va.gov](mailto:Johnathon.CobleHarrison@va.gov) 
- **VAHB leads:** @mmiddaugh @rtwell 
- **Slack Channel:** #vetride-vahb-collaboration

## Supporting Artifacts 
- [VetRide site](https://www.vetride.va.gov/app/home)

## ADE Tasks
- [ ] Collaborate `task`
- [ ] ADE Support `task`
- [ ] Document `task`
- [x] Review `task`

## Acceptance Criteria

- [ ] Review web - desktop version
- [ ] Review web - mobile version
- [ ] Provide basis based on both [CC](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric) and [508/WCAG](https://digital.va.gov/section-508/checklists/web-content-accessibility/) in case of defects

### Ticket Updates
- [ ] _Connect to an `Epic` or `Intake` (what body of work is this a part of?)_
- [ ] _Label with `Originator/Team` (product team or stakeholder requesting support; ADE if internal)_
- [ ] _Label date in the `Open Date` field_
- [ ] _Label with `Estimate` (level of effort expected for this ticket)_
- [ ] _Add `Assignee(s)` name(s) to ticket_ (include Lakwi)
- [ ] _Add `Assignee(s) name(s) to each task` they will complete via handle tag (if known)_
- [ ] _Select a `Priority Level`_
- [ ] _Update date in `Last Checked` field_
- [ ] _Label with `Actual` (level of effort it took to complete this ticket)_
- [ ] _Update date in_ `Closed Date`

**Comment by mmutia** [comment]
Work is complete. Logan Smith has distributed the COTS doc to the Veteran Transportation Team and the VetRide leadership team.
